### PR TITLE
Added Latest Accepted and Opened PRs

### PR DESCRIPTION
**Pull request by priyanka-28**

### CODE REVIEW

**Comment by darko-marinov** [comment]
Please resolve the conflicts so that we can merge your changes, ideally tonight.

File: pr-data.csv [hunk 1]
```diff
@@ -3604,6 +3610,10 @@ https://github.com/immutables/immutables,b21e6bbfbb9038100532416c81e8270951c3e03
 https://github.com/immutables/immutables,b21e6bbfbb9038100532416c81e8270951c3e03a,value-fixture,org.immutables.fixture.jackson.ObjectMappedTest.keywordNames,ID,Accepted,https://github.com/immutables/immutables/pull/1350,
 https://github.com/immutables/immutables,b21e6bbfbb9038100532416c81e8270951c3e03a,value-fixture,org.immutables.fixture.modifiable.ModifiablesTest.modifiableImmutableCollections,ID,Accepted,https://github.com/immutables/immutables/pull/1350,
 https://github.com/Infosys/olingo-odata2,daa83f4d80f3eb8019ae2cb6729486d1821b8c89,odata2-jpa-processor/jpa-core,org.apache.olingo.odata2.jpa.processor.core.access.data.JPAEntityTest.testCreateODataEntryPropertyWithOutCallBack,ID,MovedOrRenamed,,https://github.com/TestingResearchIllinois/idoft/issues/388
+https://github.com/INRIA/spoon,17330abe8aac89d907ac977e8e22ff35103eccd1,.,spoon.support.visitor.java.JavaReflectionTreeBuilderTest.testPurelyReflectiveInnerClass,ID,Opened,https://github.com/INRIA/spoon/pull/5037,
```
[the remark this answers is not in the document]
Comment (xieyt2000) [reply pr-data.csv:3613]: > 

But this change is in `testInnerClassOfSourceCodeClass`, while this line is `testPurelyReflectiveInnerClass`

[the remark this answers is not in the document]
Comment (priyanka-28) [reply pr-data.csv:3613]: Apologies, that was incorrect. Updated the PR now!

[the remark this answers is not in the document]
Comment (xieyt2000) [reply pr-data.csv:3613]: > Apologies, that was incorrect. Updated the PR now!

I haven't seen the update. Have you pushed?

[the remark this answers is not in the document]
Comment (priyanka-28) [reply pr-data.csv:3613]: The change has been made in this PR - https://github.com/INRIA/spoon/pull/5037/files

**Pull request closed without merging**